atomic_reactor/util.py

```diff
@@ -985,8 +985,93 @@ def get_manifest_list(image, registry, insecure=False, dockercfg_path=None):
     return response
 
 
+def get_all_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
+                             versions=('v1', 'v2', 'v2_list', 'oci', 'oci_index')):
```

Comment:
Let's not attempt to fetch oci and oci_index when we don't care about the results.

Reply:
ok